docs/tutorials/tutorial-sql-query-view.md

```diff
@@ -0,0 +1,168 @@
+
+   ![demo queries](../assets/tutorial-sql-demo-queries.png)
+
+3. Click the **Demo 1** tab. This query ingests sample data into a datasource called **kttm_simple**. Click the **Demo 1** tab heading again and note the options&mdash;you can rename, copy and duplicate tabs.
+
+4. Click **Run** to ingest the data.
+
+5. When ingestion is complete, Druid displays the time it took to complete the insert query, and the new datasource **kttm_simple** displays in the left pane.
+
+In this tutorial you only use the **Demo 1** query. You can explore the other tabs to see example queries that ingest, transform, and query sample data.
+
+## View and filter query results
+
+In this section you run some queries against the new datasource and perform some operations on the query results.
+
+1. Click **+** to the right of the existing tabs to open a new query tab.
+
+2. Click the name of the datasource **kttm_simple** in the left pane to display some automatically generated queries:
+
+   ![auto queries](../assets/tutorial-sql-auto-queries.png)
+
+3. Click **SELECT * FROM kttm_simple** and run the query.
+
+4. In the query results pane, click **Chrome** anywhere it appears in the **browser** column then click **Filter on: browser = 'Chrome'** to filter the results.
+
+In this tutorial you only use the **Demo 1** query. You can explore the other tabs to see examples of queries that ingest, transform, and query sample data.
+
+## Run aggregate queries
+
+In this section you run some aggregate queries and perform some operations on the query results.
+
+1. Open a new query tab.
+
+2. Click **kttm_simple** in the left pane to display the generated queries.
+
+3. Click **SELECT COUNT(*) AS "Count" FROM kttm_simple** and run the query.
+
+4. After you run a query that contains an aggregate function, additional options become available in the Query view. 
+
+   Click the arrow to the left of the **kttm_simple** datasource to display the columns, then click the **country** column. Several options appear to apply country-based filters and aggregate functions to the query:
+
+   ![count distinct](../assets/tutorial-sql-count-distinct.png)
+
+5. Click **Aggregate > COUNT(DISTINCT country)** to add this clause to the query, then run the updated query:
+
+   ![aggregate-query](../assets/tutorial-sql-aggregate-query.png)
+
+6. Click **Engine: auto (sql-native)**. From the menu that appears you can edit the query context and turn off some query defaults. 
+
+   Uncheck **Use approximate COUNT(DISTINCT)** and rerun the query. The country count in the results decreases because the computation has become more exact.
+
+7. The Query view can provide information about a function, in case you aren't sure exactly what it does.
+
+   Replace the query line `COUNT(DISTINCT country) AS dist_country` with `COUNT(DISTINCT)`. 
```

Comment:
maybe instead of "Replace" the more accurate action is "Start typing" it is only when a user types that the auto-suggestion menu pops up.

Reply:
Very true. Updated.